backend/devices/bitbox02/error.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2018 Shift Devices AG
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bitbox02
+
+const (
+	// 100 errors are reserved for errors coming from the device firmware
+	// Difference namespace should be used for local app errors
+
+	// ErrInvalidInput is returned when the request sends and invalid or unexpected input
+	ErrInvalidInput = 101
```

Comment:
those don't match the numbers in the firmware.

Reply:
Updated in this PR: https://github.com/shiftdevices/firmware_v2/pull/256/files